src/Sentry/SentryOptions.cs

```diff
@@ -436,10 +436,7 @@ public SentryOptions()
                 () => ExceptionProcessors ?? Enumerable.Empty<ISentryEventExceptionProcessor>()
             };
 
-            SentryStackTraceFactory = Runtime.Current.IsMono()
-                // Also true for IL2CPP
-                ? new MonoSentryStackTraceFactory(this)
-                : new SentryStackTraceFactory(this);
+            SentryStackTraceFactory = new SentryStackTraceFactory(this);
```

Comment:
This is the only change we need actually. Just don't add it by default on Mono but keep the code around for now until we're sure we won't need this for Unity